machines/ml/main.tf

```diff
@@ -126,13 +141,13 @@ resource "aws_ebs_volume" "controller-cache" {
 resource "aws_volume_attachment" "controller-cache-attachment" {
   instance_id = "${aws_instance.controller.id}"
   volume_id   = "${aws_ebs_volume.controller-cache.id}"
-  device_name = "/dev/sdx"
+  device_name = "${local.device_name}"
 
   skip_destroy = true
 
   provisioner "local-exec" {
     interpreter = ["/bin/bash", "-c"]
-    command = "../../scripts/run-ansible-playbook-on-host ../../services/controller/src/plz/controller/startup/startup.yml ${aws_instance.controller.private_ip} /dev/stdin <<< 'device: /dev/nvme1n1'"
+    command     = "../../scripts/run-ansible-playbook-on-host ../../services/controller/src/plz/controller/startup/startup.yml ${aws_instance.controller.private_ip} /dev/stdin <<< 'device: ${local.actual_device_name}'"
```

Comment:
What's that whitespace here?

Reply:
That’s the result of running `terraform fmt`. It aligns `=` signs.